config/manager/controller_manager_config.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: controller-runtime.sigs.k8s.io/v1alpha1
```

Comment:
shouldn't we start to drop the v1alpha1 api?

Reply:
only `v1aplha1` apis are released for controller-runtime
https://github.com/kubernetes-sigs/controller-runtime/tree/master/pkg/config